src/System.Private.CoreLib/src/System/Activator.cs

```diff
@@ -147,6 +148,51 @@ public static object CreateInstance(Type type, BindingFlags bindingAttr, Binder
         }
 
         private const BindingFlags ConstructorDefault = BindingFlags.Instance | BindingFlags.Public | BindingFlags.CreateInstance;
+
+        // https://github.com/dotnet/corefx/issues/30845
```

Comment:
Nit: We typically put the link to the issue right in front or next to the throw PNSE to make it easy to see:

`throw new PlatformNotSupportedException(); // https://github.com/dotnet/corefx/issues/30845`

It does not hurt to add the link to each throw PNSE.

Reply:
The `*From` variants do not need the link to the issue. They will throw PNSE in CoreRT by design (e.g. similar to LoadFrom).